### ISSUE TITLE
Update README.md

### ISSUE THREAD

**Issue by mtvillwock** [ideo/c-delta-challenge#12]
Not sure if this is a common occurrence, but if we want to disregard addressing styling components until later in part 2, we may want to move them farther down the `README.md` so that it doesn't leave an open loop in the candidate's mind.

Separating part 2.2 into two sections (read more link and sort by name/score buttons) might be helpful as well since they are distinct, separate features.

**Comment by geeosh** [comment]
@mtvillwock good idea, I'd defer to @msecret and @kaptron to see if they'd like to alter that part of the flow.
